rero_ils/modules/documents/listener.py

```diff
@@ -187,3 +184,27 @@ def filter_isbn(identified_by):
                 isbns.add(to_isbn10(isbn))
         if isbns:
             json['isbn'] = list(isbns)
+
+
+def update_collections(item_pids, json, agg_size=1000):
+    """Add all collection titles and descriptions.
```

Comment:
Add to the document index?

Maybe it's perfectly clear with the context, I don't know.